app/src/main/java/com/mobiledevelopmentfoodapp/dto/Order.kt

```diff
@@ -1,10 +1,14 @@
 package com.mobiledevelopmentfoodapp.dto
 
-//import androidx.room.Entity
-//import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Order(var customerId: Int, var name: String, var products: ArrayList<Food>, var totalPrice: Double, @SerializedName("id") var orderId:Int = 0 ) {
+data class Order(
+    internal var customerId: Int,
+    internal var name: String,
+    internal var products: ArrayList<Food>,
+    internal var totalPrice: Double,
+    @SerializedName("id")internal var orderId: Int = 0,
```

Comment:
Added trailing comma. A trailing comma is a stylistic choice. The Kotlin style guide recommends a trailing comma. Best practices for trailing commas can be found here. https://kotlinlang.org/docs/coding-conventions.html#trailing-commas